language/de

```diff
@@ -148,7 +148,13 @@
 			"swap": "Change this Page"
 		}
 	},
-	"page_delete_alert": "This will delete the current design page.<br/>Do you wish to continue?",
-	"gallery_search": "Type to search gallery..",
-	"save_later": "Save for Later"
+	"page_delete_alert": "Dies löscht die aktuelle Design-Seite. <br/>Wollen Sie fortfahren?",
+	"gallery_search": "In der Galerie suchen..",
+	"save_later": "Für später speichern"
```

Comment:
add comma here.

Reply:
Look Guys you are getting free translation for your product and send us the and or remove comma, instead of doing it quickly yourself ..... 

Not sure it that is wise to do.... 👎

Reply:
Hey thanks for the translation and we have added and removed the commas, it's done on our side and updated on our server. This is just so that we can merge your pull request. See here: http://pitchprint.io/rsc/lang/de

Reply:
Ahh ok... Cool... I see the changes have already gone through. Cheers